open-api-spec.yaml

```diff
@@ -1328,6 +1299,14 @@ components:
     '403':
       description: Forbidden
   parameters:
+    id:
+      name: id
+      in: path
+      schema:
+        type: string
+        format: uuid
```

Comment:
Yes or no on enforcing uuid?
https://github.com/interledger/rafiki/pull/603#discussion_r975017871

Reply:
I vote no. That's a business decision.